.circleci/config.yml

```diff
@@ -501,7 +521,15 @@ workflows:
             branches:
               only:
                 - '/^release-0\.\d+$/'
-
+      - release_bom:
+          context:
+            - github-repo
+            - gcp-astronomer-prod
+          filters:
+              branches:
+                only:
+                  - '/^release-0\.\d+$/'
+                  - 'fix-6117-adding-ci-job'
```

Comment:
We should change this to only run on tags, not on branches. If we do that, then we can use the CIRCLECI env var that shows which tag triggered the run, `CIRCLE_TAG` <https://circleci.com/docs/variables/>

Reply:
sure, added the tag for now i am keeping the branches as well for testing, @danielhoherd can you explain how we can use CIRCLE_TAG.? Should it be put as an echo $CIRCLE_TAG .?

Reply:
CIRCLE_TAG didn't work but awk '/^version: / {print $2}' Chart.yaml works fine @danielhoherd

Reply:
CIRCLE_TAG is not going to work unless it's running from a tag, so it won't work in this branch.